### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by jyotsnaven**
Please review my assignment!
@jpang-h4  @duaneatat @mikwat @ali-tetration

### CODE REVIEW

File: assignment4/src/routes.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    .config(RoutesConfig);
+
+  RoutesConfig.$inject = ['$stateProvider', '$urlRouterProvider'];
+  function RoutesConfig($stateProvider, $urlRouterProvider) {
+
+    $urlRouterProvider.otherwise('/');
+
+    $stateProvider
+
+    .state('home', {
```
Comment (duaneatat) [inline assignment4/src/routes.js]: nit - indent this continuation

File: assignment4/src/menuapp/menuapp.service.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+
+(function() {
+
+  angular
+    .module('MenuApp')
+    .service('MenuAppService', MenuAppService)
+    .constant('ApiURL_categories', 'http://davids-restaurant.herokuapp.com/categories.json')
```
Comment (duaneatat) [inline assignment4/src/menuapp/menuapp.service.js]: nit - constants in `UPPERCASE_UNDERSCORES`

File: assignment4/src/menuapp/itemslist.component.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+
+(function() {
+
+  angular
+    .module('MenuApp')
+    .component('itemsList', {
```
Comment (duaneatat) [inline assignment4/src/menuapp/itemslist.component.js]: nice! 👍 at some point we'll upgrade angular to a version that supports components.

Comment (jyotsnaven) [inline assignment4/src/menuapp/itemslist.component.js]: Thank you Duane! Made changes and merged.

**Pull request merged**